<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="Frame_FondoMensajeria">
      <property name="styleSheet">
       <string notr="true">#Frame_FondoMensajeria {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #cde5fa, stop:0.5 #eaf6ff, stop:1 #ffffff);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,5">
       <item>
        <widget class="QFrame" name="Frame_TituloMensajeria">
         <property name="styleSheet">
          <string notr="true">    background: transparent;
    border: none;  /* Opcional: elimina el borde si tambi√©n lo quieres transparente *</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>20</y>
            <width>311</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Franklin Gothic Book</family>
            <pointsize>20</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Windows Live Messenger</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>81</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>Logo_Windows_Live_Messenger(2).png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="Frame_FondoPrincipal">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>20</y>
               <width>101</width>
               <height>16</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Franklin Gothic Book</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Hola: Usuario123!</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>40</y>
               <width>131</width>
               <height>121</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(197, 197, 197);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QPushButton" name="Boton_Perfil">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>180</y>
               <width>131</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Franklin Gothic Book</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 170, 255);</string>
             </property>
             <property name="text">
              <string>Mi Perfil</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::AddressBookNew"/>
             </property>
            </widget>
            <widget class="QPushButton" name="Boton_ChatG">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>210</y>
               <width>131</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Franklin Gothic Book</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 255, 0);</string>
             </property>
             <property name="text">
              <string>Chat General</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::GoHome"/>
             </property>
            </widget>
            <widget class="QPushButton" name="Boton_NuevoChat">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>240</y>
               <width>131</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Franklin Gothic Book</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 170, 0);</string>
             </property>
             <property name="text">
              <string>Nuevo Chat</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::ContactNew"/>
             </property>
            </widget>
            <widget class="QPushButton" name="Boton_Grupos">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>270</y>
               <width>131</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Franklin Gothic Book</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 40, 30);</string>
             </property>
             <property name="text">
              <string>Nuevo Grupo</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::NetworkOffline"/>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_5">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>360</y>
               <width>131</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Terminal</family>
               <pointsize>9</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 85, 255);</string>
             </property>
             <property name="text">
              <string>Aplicaciones</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::EditSelectAll"/>
             </property>
            </widget>
            <widget class="QPushButton" name="Boton_Grupos_2">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>300</y>
               <width>131</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Franklin Gothic Book</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 15, 135);</string>
             </property>
             <property name="text">
              <string>Grupos</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::NetworkOffline"/>
             </property>
            </widget>
            <widget class="QPushButton" name="Boton_Grupos_3">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>330</y>
               <width>131</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Franklin Gothic Book</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(155, 5, 0);</string>
             </property>
             <property name="text">
              <string>Salir de grupo</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::NetworkOffline"/>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <widget class="QPlainTextEdit" name="plainTextEdit">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>30</y>
               <width>471</width>
               <height>331</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(147, 147, 147);</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>370</y>
               <width>381</width>
               <height>21</height>
              </rect>
             </property>
            </widget>
            <widget class="QPushButton" name="Boton_Enviar">
             <property name="geometry">
              <rect>
               <x>400</x>
               <y>370</y>
               <width>75</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Franklin Gothic Book</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Enviar</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
             </property>
            </widget>
            <widget class="QPushButton" name="Boton_Zumbido">
             <property name="geometry">
              <rect>
               <x>410</x>
               <y>0</y>
               <width>75</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Franklin Gothic Book</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Timbre</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::CallStart"/>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
